### ISSUE TITLE
Add Aleph.im to Decentralized Storage page

### ISSUE THREAD

**Issue by adrien-be** [ethereum/ethereum-org-website#2240]
**Product name:**

[Aleph.im](https://aleph.im)

**Product description:**

Decentralized cloud project (database, file storage, computing and DID). A unique blend of offchain and onchain peer-to-peer technology. It is already supporting Ethereum, Polkadot, Cosmos and BinanceChain. Aleph.im is compatible with IPFS. The network is live, staking was launched in Q4 2020. There are already several DApps running on the network.

**How long has it been around?**

Moshe, the founder, started developing aleph.im in 2018.

**Where do you think it should go?**

https://ethereum.org/en/developers/docs/storage/

**GitHub URL**

[GitHub](https://github.com/aleph-im/)

**Docs URL**

[Documentation](https://aleph.im/#/developers)

**Is the product open source?**

Yes. Aleph.im is 100% open source.

**Additional context**

**Comment by ryancreatescopy** [comment]
@wackerow we should take a look at this asap

**Comment by adrien-be** [comment]
Thank you @ryancreatescopy 

Note that we just added support for Avalanche and Solana.

I forgot to mention that our network is already live since Dec 2020 and we had a 120% growth in validators during our first 6 weeks only. We currently have 37 validators and expect the growth to keep going as we successfully got through the toughest part of a network effect: the first weeks of the launch, igniting "the network effect" in the figurative and literal meaning.
